doc/policies/system.rst

```diff
@@ -1,52 +0,0 @@
-.. _system_policies:
```

Comment:
I guess this page is obsolete now :-) Do I understand correctly that the system scope simply does not exist anymore?

Reply:
Yes. The system scope is a remnant von LinOTP and privacyIDEA 1.5.
Writing configurations was in the system scope. This was all moved as different actions to the admin scope.